src/Website/Views/ContactUs/ContactUs.cshtml

```diff
@@ -0,0 +1,24 @@
+﻿
+@{
+    ViewBag.Title = "Contact Us";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+<article>
+    <header>
+        <hgroup>
+            <h1>The Westman Alers</h1>
+        </hgroup>
+    </header>
+    <section>
+        <h2>Who We Are</h2>
+        <p>Description of the club, its activities and goals go here.</p>
+    </section>
+    <section>
+        <h2>How To Contact Us</h2>
+        <p>Email address goes here.</p>
+    </section>
+</article>
+
```

Comment:
Crossed my mind too, but it felt weird since it would either be it's own table with only ever one record in it or be stored in some non normalized catch all table.

Unless you have any strong objections I vote for leaving it as is for now.  Maybe when we dig into some of the other pages we'll think of something that could also be applied here.